<ui version="4.0" >
 <class>Login</class>
 <widget class="QDialog" name="Login" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>登录</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_login" >
     <property name="title" >
      <string>登录</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_name" >
        <property name="text" >
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit_name" >
        <property name="text" >
         <string/>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_pass" >
        <property name="text" >
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_pass" >
        <property name="text" >
         <string/>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_reg" >
     <property name="title" >
      <string>注册并登录</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_regname" >
        <property name="text" >
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit_regname" >
        <property name="maxLength" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_regpass" >
        <property name="text" >
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_regpass" >
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_regpass2" >
        <property name="text" >
         <string>再输一遍密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit_regpass2" >
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_ip" >
     <property name="text" >
      <string>服务器IP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lineEdit_ip" >
     <property name="text" >
      <string/>
     </property>
     <property name="maxLength" >
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pushButton_login" >
     <property name="text" >
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="pushButton_reg" >
     <property name="text" >
      <string>注册</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
